python/ray/tune/trial_runner.py

```diff
@@ -618,6 +436,8 @@ def add_trial(self, trial):
         self.trial_executor.try_checkpoint_metadata(trial)
 
     def debug_string(self, delim="\n"):
+        from ray.tune.progress_reporter import trial_progress_str
```

Comment:
avoid circular imports. also makes sense here because debug_string is just here for, well, debugging